apps/src/p5lab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -75,23 +93,60 @@ class AnimationPicker extends React.Component {
     } else if (this.props.uploadInProgress) {
       return <h1 style={styles.title}>{msg.animationPicker_uploading()}</h1>;
     }
+
+    // Update text depending on the context of the animation picker
+    let imageText;
+    switch (this.props.pickerType) {
+      case PICKER_TYPE.spritelab:
+        imageText = 'costumes';
+        break;
+      case PICKER_TYPE.gamelab:
+        imageText = 'animations';
+        break;
+      default:
+        imageText = 'backgrounds';
+    }
+
     return (
-      <AnimationPickerBody
-        is13Plus={this.props.is13Plus}
-        onDrawYourOwnClick={this.props.onPickNewAnimation}
-        onPickLibraryAnimation={this.props.onPickLibraryAnimation}
-        onUploadClick={this.onUploadClick}
-        onAnimationSelectionComplete={this.props.onAnimationSelectionComplete}
-        playAnimations={this.props.playAnimations}
-        libraryManifest={this.props.libraryManifest}
-        hideUploadOption={this.props.hideUploadOption}
-        hideAnimationNames={this.props.hideAnimationNames}
-        navigable={this.props.navigable}
-        defaultQuery={this.props.defaultQuery}
-        hideBackgrounds={this.props.hideBackgrounds}
-        canDraw={this.props.canDraw}
-        selectedAnimations={this.props.selectedAnimations}
-      />
+      <div>
+        <AnimationPickerBody
+          is13Plus={this.props.is13Plus}
+          onDrawYourOwnClick={this.props.onPickNewAnimation}
+          onPickLibraryAnimation={this.props.onPickLibraryAnimation}
+          onUploadClick={this.onUploadClick}
+          onAnimationSelectionComplete={this.props.onAnimationSelectionComplete}
+          playAnimations={this.props.playAnimations}
+          libraryManifest={this.props.libraryManifest}
+          hideUploadOption={this.props.hideUploadOption}
+          hideAnimationNames={this.props.hideAnimationNames}
+          navigable={this.props.navigable}
+          defaultQuery={this.props.defaultQuery}
+          hideBackgrounds={this.props.hideBackgrounds}
+          canDraw={this.props.canDraw}
+          selectedAnimations={this.props.selectedAnimations}
+        />
+        {this.state.exitingDialog && (
+          <BaseDialog isOpen hideCloseButton={true} style={styles.dialog}>
```

Comment:
this dialog should be its own component. even though it likely won't be reused, it'll make AnimationPicker more readable

Reply:
nit: any boolean prop that's set to `true` can be simplified the way `isOpen` is:

```javascript
<MyComponent
  isOpen={true}
  isOpen  // means the same as the line above
/>
```

Reply:
could we also use StylizedBaseDialog here? it uses BaseDialog under the hood, so you can still use props like `hideCloseButton`. i think it would look most similar to the usage in [InstructionsDialog](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/instructions/InstructionsDialog.jsx)